bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/handler/HomematicThingHandler.java

```diff
@@ -420,6 +419,37 @@ protected void updateDatapointState(HmDatapoint dp) {
         }
     }
 
+    private Object getValueForConfiguration(HmDatapoint dp) {
+        if (dp.getValue() == null) {
+            return null;
+        }
+        if (dp.isEnumType()) {
+            return dp.getOptionValue();
+        }
+        if (dp.isNumberType()) {
+            // For number datapoints that are only used depending on the value of other datapoints,
+            // the CCU may return invalid (out of range) values if the datapoint currently is not in use.
+            // Make sure to not invalidate the whole configuration by returning the datapoint's default
+            // value in that case.
+            final boolean minValid, maxValid;
+            if (dp.isFloatType()) {
+                Double numValue = dp.getDoubleValue();
+                minValid = dp.getMinValue() == null || numValue >= dp.getMaxValue().doubleValue();
+                maxValid = dp.getMinValue() == null || numValue <= dp.getMinValue().doubleValue();
```

Comment:
It looks to me wrong: the first should I believe consider dp.getMinValue() while the second should consider dp.getMaxValue(). WDYT ?
Same remark in the else branch

Reply:
Yes, you're completely right. Not sure what I was thinking there, probably copy'n'paste gone wrong. Fixed now.